src/components/projectcard.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ProjectCard = ({ image, title, text, color }) => (
+  <div className={`project-card ${color}`}>

```

Comment:
instead of setting color by props, you can use [nth-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) pseudo selector in css